test/e2e-multicluster-test/testdata/app/app-with-fixed-location.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: app
+spec:
+  components:
+    - type: k8s-objects
+      name: app
+      properties:
+        objects:
+          - apiVersion: v1
+            kind: ConfigMap
+            metadata:
+              name: "x"
+              labels:
+                app.oam.dev/cluster: local
```

Comment:
So this label will override the cluster in topology, right?
If we write a workflow step like:
```
resource: op.#Apply & {
  cluster: "test"
  value: {
            apiVersion: v1
            kind: ConfigMap
            metadata:
              name: "x"
              labels:
                app.oam.dev/cluster: local
  }
}
```
The config map will be applied to the local cluster, right?

Reply:
Right.